docs/templates.md

```diff
@@ -18,6 +18,7 @@ The following keywords can be used in `jj log`/`jj obslog` templates.
 * `description: String`
 * `change_id: ChangeId`
 * `commit_id: CommitId`
+* `parent_commit_ids: List<CommitId>`
```

Comment:
`working_copies`, `branches`, `tags` and `git_refs` should be `List<String>` then too

Reply:
Yep. Added TODO comments.